src/lib/__tests__/AnnotationControlsFSM-test.js

```diff
@@ -59,13 +59,21 @@ describe('lib/AnnotationControlsFSM', () => {
                 expect(annotationControlsFSM.getState()).to.equal(AnnotationState.NONE);
             });
         });
+
+        // Should reset state
+        it('should reset state if input is AnnotationInput.RESET', () => {
```

Comment:
The tests are grouped with starting state. We have three large groups (`describe('AnnotationState.NONE'`, `describe('AnnotationState.HIGHLIGHT/REGION'` and `describe('AnnotationState.HIGHLIGHT_TEMP/REGION_TEMP'`)
You introduced a new input, so you should add one test under each of above three groups.